third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h

```diff
@@ -85,6 +85,13 @@ unsigned getVectorSize(Value ptr, Value offset,
 
 Type scaleDotElemTypeToMLIRType(MLIRContext *ctx, triton::ScaleDotElemType t);
 
+// Return true if the src encoding and the destination encoding (in LDS) result
```

Comment:
"Returns ture if we can perform coalesced write from the source encoding to the destination encoding."